build/circle-trigger-nightly.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -eu
+
+REV=$(git describe --tags)
```

Comment:
It's weird that this runs the build on whatever happens to be checked out. Do we need to determine the revision locally instead of just building `master`? (we could turn off the deployment stuff based on the `NIGHTLY` variable). If we do need to pass a revision in to circleci then we could do something like `git fetch origin && git describe --tags origin/master`.